src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let money = days * 40;
+
+  if (days >= 7) {
+    money = money - 50;
```

Comment:
Replace the literals '7' and '50' with named constants to clarify their meaning. For example, 'const LONG_TERM_RENTAL = 7;' and 'const LONG_TERM_DISCOUNT = 50;'.